logstash/templates/configmap-config.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.logstashConfig }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}-config
+  labels:
+    app: "{{ template "fullname" . }}"
+    chart: "{{ .Chart.Name }}"
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+data:
+{{- range $path, $config := .Values.logstashConfig }}
+  {{ $path }}: |
+{{- if (eq $path "logstash.yml") }}
+    node.name: {{ template "fullname" $ }}
```

Comment:
This is going to be the release name which won't be unique per replica in the statefulset. 

Looking at https://www.elastic.co/guide/en/logstash/current/logstash-settings-file.html it just defaults to the hostname so I think we can just leave it out. 

If it isn't auto detecting things for some reason I think we could do something like https://github.com/elastic/helm-charts/blob/340868c54c44ae97fd9ef8fa6d682472098dc816/elasticsearch/templates/statefulset.yaml#L334-L338

But instead map it to a different environment variable that Logstash doesn't run env2yaml on and refer to it with something like:

`node.name $OTHER_VAR_NAME` in the default config

Reply:
AFAIK, `node.name` using `hostname` as default value should be OK.
Fixed in https://github.com/elastic/helm-charts/pull/333/commits/cad113038b875e0d1db2dd548a26dc6cb190bd85.